packages/connect-iframe/src/index.ts

```diff
@@ -233,13 +231,6 @@ const postMessage = (message: CoreEventMessage) => {
         return;
     }
 
-    if (message.event === TRANSPORT_EVENT) {
```

Comment:
In any case, shouldn't this still be here? Maybe not if it was used only in suite.